arches_querysets/models.py

```diff
@@ -247,10 +243,10 @@ def save(self, *, request=None, index=True, **kwargs):
                 pk=models.F("nodegroup")
             )
         request = request or self._request
-        with transaction.atomic():
-            if self.sortorder is None or self.is_fully_provisional():
-                self.set_next_sort_order()
-            self._save_aliased_data(request=request, index=index, **kwargs)
+        # Mimic some computations trapped on TileModel.save().
+        if self.sortorder is None or self.is_fully_provisional():
+            self.set_next_sort_order()
+        self._save_aliased_data(request=request, index=index, **kwargs)
```

Comment:
@chrabyrd I'm hoping this fixes #54, can you confirm?

Keeping transactions as short as possible is a good practice, and `TileTreeOperation._save()` was already using one, so this one was too wide.